source/common/router/route_config_update_receiver_impl.cc

```diff
@@ -78,10 +81,11 @@ bool RouteConfigUpdateReceiverImpl::onVhdsUpdate(
     const VirtualHostRefVector& added_vhosts, const std::set<std::string>& added_resource_ids,
     const Protobuf::RepeatedPtrField<std::string>& removed_resources,
     const std::string& version_info) {
-
   auto vhosts_after_this_update =
-      std::make_unique<std::map<std::string, envoy::config::route::v3::VirtualHost>>(
-          *vhds_virtual_hosts_);
+      vhds_virtual_hosts_ == nullptr
```

Comment:
WDYT of making this a normal conditional rather than a ternary.

It probably doesn't make a difference in terms of what it compiles to, but doing that would have 2 (minor advantages):  slightly easier to read (IMO), and it would be easier to verify that you hit both branches in tests by looking at coverage.

Reply:
Done